Algorithm-Caesars-Cipher.md

```diff
@@ -1,29 +1,27 @@
-:triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](http://github.com/FreeCodeCamp/FreeCodeCamp/wiki/How-to-get-help-when-you-get-stuck) if you get stuck. Try to pair program. :busts_in_silhouette: Write your own code. :memo:
+:triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](http://github.com/FreeCodeCamp/FreeCodeCamp/wiki/How-to-get-help-when-you-get-stuck) if you get stuck. Try to pair program :busts_in_silhouette: . Write your own code :memo: .
```

Comment:
I would put the icon at the beginning and the dot at the end. It doesn't looks good the way it is with the dot after the icon. Too much space.